src/main/java/mate/controller/AddDriverController.java

```diff
@@ -0,0 +1,32 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.DriverService;
+
+public class AddDriverController extends HttpServlet {

```

Comment:
```suggestion
@WebServlet
public class AddDriverController extends HttpServlet {

```

lets use @WebServlet annotation everywhere or nowhere, so our code looks consistent